### PR TITLE
Update scroll handling on schemaform pages

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to [1778](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1778)

Adds back scrolling on FormPage and sets scroll restoration for manual to keep Chrome/FF from jumping to top of page and scrolling down.

### CODE REVIEW

**Review by cvalarida — APPROVED** [review]
Running OS X, I wasn't able to test it on IE, but it works great on Chrome and Firefox.

**Comment by cvalarida** [comment]
Any idea what's up with that failed e2e test?

File: src/js/common/schemaform/FormPage.jsx [hunk 1]
```diff
@@ -17,6 +18,15 @@ function focusForm() {
   }
 }
 
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', window.VetsGov.scroll || {
```
Comment (cvalarida) [inline src/js/common/schemaform/FormPage.jsx]: So...documentation isn't a silver bullet or anything--especially when you amass too much of it--but I wonder if this is the type of thing that we should have written somewhere.

"`window.VetsGov` is a thing! This is what it does."

Comment (jbalboni) [inline src/js/common/schemaform/FormPage.jsx]: Yeah. I didn't realize it was still used for things. It used to be used to swap urls out for e2e tests, but I think we replaced that with the environment.js file.

**Pull request merged**